.circleci/config.yml

```diff
@@ -7,10 +7,10 @@ jobs:
   build:
     working_directory: /home/circleci/app
     docker:
-      - image: cimg/python:3.7.14-node
+      - image: cimg/python:3.10.10-node
         environment:
           SQLALCHEMY_DATABASE_URI: postgresql://taskingmanager@localhost/test_tm
-      - image: cimg/postgres:14.2-postgis
+      - image: cimg/postgres:15.2-postgis
```

Comment:
Is this version of Postgresql tested?

Reply:
I don't know how relevant this is, but PostgreSQL 15 has now security restricted the usage of public schema willy-nilly. It's a good thing but we just have to make sure it doesn't affect the app. I would recommend staying with the latest v14 which currently is v14.7. 

I will defer to @Aadesh-Baral for the final decision.

Reply:
In my testing, this did not appear to be an issue.

Specifically, I ran the tests against a database created with the following parameters:
* Docker images: `cimg/postgres:15.2-postgis` and `postgis/postgis:latest`
* POSTGRES_PASSWORD=hottm
* POSTGRES_USER=hottm
* POSTGRES_DB=test_tasking-manager

Reply:
Let's use 14.7 for the time being and upgrade to v15 later after investigating the issue @eternaltyro mentioned.

Reply:
I've moved back to 14.7 for now.